cmd/migrate-app-ids/main.go

```
@@ -0,0 +1,275 @@
+			[]interface{}{uuid, id.orgHandle, id.clientID}},
+		{`UPDATE application_data SET app_id = $1
+			WHERE app_id = $2 AND profile_id IN (SELECT profile_id FROM profiles WHERE org_handle = $3)`,
+			[]interface{}{uuid, id.clientID, id.orgHandle}},
+		{`UPDATE consent_category_attributes SET application_identifier = $1
+			WHERE application_identifier = $2
+			AND category_id IN (SELECT category_identifier FROM consent_categories WHERE org_handle = $3)`,
+			[]interface{}{uuid, id.clientID, id.orgHandle}},
+	}
+
+	for _, s := range stmts {
+		if _, err := tx.Exec(s.query, s.args...); err != nil {
```

golangci-lint (2.12.2)

[error] (*database/sql.Tx).Exec must not be called. use (*database/sql.Tx).ExecContext

(noctx)